cerberus-audit-logger-athena/src/test/java/com/nike/cerberus/audit/logger/service/AthenaServiceTest.java

```diff
@@ -0,0 +1,40 @@
+package com.nike.cerberus.audit.logger.service;
+
+import static junit.framework.Assert.assertTrue;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+import com.nike.cerberus.audit.logger.AthenaClientFactory;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+public class AthenaServiceTest {
+
+  AthenaClientFactory athenaClientFactory = new AthenaClientFactory();
+  @Mock private AthenaService mockAthenaService;
+  private AthenaService athenaService;
+
+  @Before
+  public void before() {
+    initMocks(this);
+    athenaService = new AthenaService("fake-bucket", athenaClientFactory);
+  }
+
+  @Test
+  public void test_that_addPartition_works() {
+    String fileName = "localhost-audit.2018-01-29_12-58.log.gz";
+    try {
+      athenaService.addPartitionIfMissing("us-west-2", "fake-bucket", "2018", "01", "29", "12");
+    } catch (Exception e) {
+      assertTrue(false);
+    }
+    assertTrue(true);
```

Comment:
Wait, what does this do? You are asserting that true == true? What is this testing?

Reply:
The method being called should be executed smoothly and if at all an exception occurs it is  not expected. This is test for void method and actually not required a junit.

Reply:
i will remove the assert true, it's by mistake, i did not notice